eth/stagedsync/stage_execute.go

```diff
@@ -570,6 +579,10 @@ Loop:
 		}
 	}
 
+	if err = diagnostics.Send(diagnostics.AppendBlockMetrics{ExecutionStart: executionStartTimes, ExecutionEnd: executionEndTimes}); err != nil {
```

Comment:
Is there a need for us to see both start times and end times?
Would it not be a more useful metric to calculate "execution durations" (end-start)?
Maybe you can still send start and end times to diagnostics but maybe diagnostics should do the calculation when it receives this?

Reply:
resolved offline - all good here just need to improve naming